snippets/language-asciidoc.cson

```diff
@@ -44,10 +44,10 @@
       ----
       $0
       '''
-  'Code Block':
-    description: 'Code Block'
+  'Source Block':
+    description: 'Source block'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#listing-blocks'
-    prefix: '--s'
+    prefix: '``'
```

Comment:
Good point. I'm still trying to find the right way to express this. However, I think generally you don't use the markdown fenced code block if you use the source block, so we'd be encouraging one over the other here. I'm open to other ideas of how to get it down to two characters.